cmake/modules/FindMaya.cmake

```diff
@@ -361,6 +361,15 @@ if (MAYA_INCLUDE_DIRS AND EXISTS "${MAYA_INCLUDE_DIR}/maya/MSelectionContext.h")
     endif()
 endif()
 
+set(MAYA_HAS_CRASH_DETECTION FALSE CACHE INTERNAL "isInCrashHandler")
+if(MAYA_INCLUDE_DIRS AND EXISTS "${MAYA_INCLUDE_DIR}/maya/MGlobal.h")
+    file(STRINGS ${MAYA_INCLUDE_DIR}/maya/MGlobal.h MAYA_HAS_API REGEX "isInCrashHandler")
+    if(MAYA_HAS_API)
+        set(MAYA_HAS_CRASH_DETECTION TRUE CACHE INTERNAL "isInCrashHandler")
+        message(STATUS "Maya has isInCrashHandler API")
```

Comment:
Hi @pierrebai-adsk , I am looking at Maya source code and no where can I find "isInCrashHandler"? When was this flag added?

Reply:
@pierrebai-adsk nvm my previous comment. I just saw your Maya PR.  Thanks